src/utilities/useLyra.js

```diff
@@ -10,3 +10,16 @@ export const useLyra = () => {
   });
   return hasLyra;
 };
+
+export const lyraGetPublicKey = async (onUpdate) => {
+  try {
+    const lyraResponse = await getPublicKey();
+
+    if (lyraResponse.error) {
+      return lyraResponse.error;
+    }
```

Comment:
this is fine for this version - but just FYI, the version of lyra-api i'm about to release will properly `throw` an error if it can't find a public key rather than returning `{ error: string }`

Reply:
@piyalbasu updated!